e2e/cypress/common/apiCalls/testData/testData.ts

```diff
@@ -126,13 +126,14 @@ export type PermissionsOptions = {
 };
 
 export const generatePermissionsData = {
-  generate: (options: Partial<PermissionsOptions>) => {
+  generate: (options: Partial<PermissionsOptions>, useNamespaces = false) => {
```

Comment:
This was tricky, because everything in `PermisionsOptions` is iterable and it gets iterated over below.

I tried various approaches and wasn't successful, so I decided to use a new parameter.

Reply:
I think you should be able to add it as an iterable as well, it's iterable because generally the searchParams can have multiple values, but I would reuse the options object that is already there.